pkg/scheduler/eventhandlers.go

```diff
@@ -466,6 +466,8 @@ func addAllEventHandlers(
 			AddFunc: sched.onStorageClassAdd,
 		},
 	)
+
+	informerFactory.Policy().V1beta1().PodDisruptionBudgets().Lister()
```

Comment:
@liggitt how do we make sure listers are properly initialized? It seems that we have to do something like this at scheduler initialization, or is the pattern that we have to pass a lister around instead of the sharedInformer instance itself?

Reply:
@Huang-Wei what we can do is store a reference to the lister at the preemption plugin initialization.

Reply:
> @Huang-Wei what we can do is store a reference to the lister at the preemption plugin initialization.

That's doable, although it sounds a bit tricky as it can be retrieved from the SharedInformerFactory. Let's first check with api experts to see the best practice here.

Reply:
you cannot assume the PDB API is available (it is a beta API that can be disabled). this needs to tolerate it not being served and not block/fail startup

Reply:
I see. Now it's being wrapped in a featuregate checking block.

Reply:
So @liggitt, back to the question, can we put a bare `informerFactory....Lister()` to trigger the lister initialization? If not, is there a better way?

(the context is we don't need a pdbInformer to watch it; we just use the pdbLister to get the pdb list from the cache storage)

Reply:
> can we put a bare informerFactory....Lister() to trigger the lister initialization

Sure. Doing so once (only if the feature is enabled) and saving the result is what we typically do. That's what the code did previously.